--- a/cmake_aux/Modules/ARMA_FindOpenBLAS.cmake	2019-12-07 01:08:22.352240900 +0300
+++ b/cmake_aux/Modules/ARMA_FindOpenBLAS.cmake	2019-12-07 01:09:48.183172700 +0300
@@ -2,6 +2,7 @@
 set(OpenBLAS_NAMES ${OpenBLAS_NAMES} openblaso)
 set(OpenBLAS_NAMES ${OpenBLAS_NAMES} openblasp)
 set(OpenBLAS_NAMES ${OpenBLAS_NAMES} openblas )
+set(OpenBLAS_NAMES ${OpenBLAS_NAMES} openblas_d)
 
 set(OpenBLAS_TMP_LIBRARY)
 set(OpenBLAS_TMP_LIBRARIES)
@@ -10,7 +11,7 @@
 foreach (OpenBLAS_NAME ${OpenBLAS_NAMES})
   find_library(${OpenBLAS_NAME}_LIBRARY
     NAMES ${OpenBLAS_NAME}
-    PATHS ${CMAKE_SYSTEM_LIBRARY_PATH} /lib64 /lib /usr/lib64 /usr/lib /usr/local/lib64 /usr/local/lib /opt/local/lib64 /opt/local/lib /usr/lib/openblas/ /usr/lib/openblas/lib /usr/local/opt/openblas/lib /opt/local/lib/openblas /opt/local/lib/openblas/lib
+    PATHS ${CMAKE_SYSTEM_LIBRARY_PATH} ${CONAN_OPENBLAS_ROOT}/lib /lib64 /lib /usr/lib64 /usr/lib /usr/local/lib64 /usr/local/lib /opt/local/lib64 /opt/local/lib /usr/lib/openblas/ /usr/lib/openblas/lib /usr/local/opt/openblas/lib /opt/local/lib/openblas /opt/local/lib/openblas/lib
     )
   
   set(OpenBLAS_TMP_LIBRARY ${${OpenBLAS_NAME}_LIBRARY})
